bip78/src/fee_rate.rs

```diff
@@ -46,6 +46,14 @@ impl Mul<FeeRate> for u64 {
     }
 }
 
+impl Div<Weight> for FeeRate {
+    type Output = FeeRate;
```

Comment:
The unit doesn't make sense. `FeeRate` = `Amount` / `Weight`, so `FeeRate` / `Weight` should be `Amount` / (`Weight` ^ 2) and there's no such unit in Bitcoin.